quinn-proto/src/frame.rs

```diff
@@ -35,10 +24,10 @@ impl Type {
 
 impl coding::Codec for Type {
     fn decode<B: Buf>(buf: &mut B) -> coding::Result<Self> {
-        Ok(Type(buf.get()?))
+        Ok(Type(buf.get_var()?))
```

Comment:
Maybe add some todo about enforcing the minimal encoding of varints here?

Reply:
I went ahead and just added the check. Good catch!